source/data/tier1data/features/group_administration.rst

```diff
@@ -0,0 +1,125 @@
+
+
+
+******************
+Managing subgroups
+******************
+
+Subgroups are used to gives permissions to a specific group of members from your project.
+Managers can create and manage subgroups by going to the Group Administration tab of the ManGO portal.
+
+After selecting your project, you will see an overview of all subgroups. 
+If they have a lock sign next to them, these are managed by the platform and can be viewed, but not edited.
+Examples include:
+
+- The main group of your project  
+- The 'managers' group  
+- ...
+
+Other groups, which you can create yourself, are called 'custom groups'.
+For example, you could create a custom group for staff and one for master students. 
+To do so, click on 'Add group' and give a name.
+Custom groups will always get your project name as prefix. For example, if your project is called 'demo-project' and you enter 'master_students', the custom group will be called 'demo-project_master_students'.
+
+When you create a new group, or where you click on one, you will get an overview of the group members on the left side.
+On the right side, you will see the members of the project who are not yet part of this group.  
+
+
+.. figure:: ../images/group_administration/mango_portal_group_administration.png
+  :alt: Editing the members of a custom group
+
+To add a member to the group, select their checkbox and click on 'Add selected users'.
+
+Likewise, to remove users from the group, select their checkbox and click on 'Remove selected users'.
+
+**Note:** You may need to log out and login again before you see changes you made in groups reflected when editing permissions. 
+
+
+
+Help! I want to collaborate with an external researcher
+-------------------------------------------------------
+
+As demonstrated earlier, you can add collaborators to your group if they have a VSC account
+If you want to work together with someone who doesn't have a VSC account, there are two options:
+
+1) Applying for a VSC account
+
+You can let your collaborators apply for a VSC account.
+This is the best solution if you have a small group of close collaborators.
+When they have a VSC-account, externals can get access to all functionalities of Tier-1 Data.
+
+For more information about requesting a VSC account, see :ref:`apply for account`.
+
+2) Sharing data via Globus
+
+:ref:`Globus<globus platform>` is a tool which allows you to transfer large datasets and share them with externals.
+To log in to Globus, users just need one of the following:
+
+- an institutional account from any institute that is connected to Globus
+- an ORCID ID 
+- a Gmail account
+
+Via a so-called guest collection, you can share data from different storage systems -including Tier-1 Data- with externals.
+You can give users or groups of users either read or write access on your data.
+However, there are some caveats:
+
+- Users will only be able to download or upload data via the Globus interface, and not via any of the other Tier-1 Data clients.

```

Comment:
`Tier-1 Data Clients` should be a link to the section in VSC docs where this clients are presented.